packages/launch/src/hooks/index.ts

```diff
@@ -0,0 +1,8 @@
+// this whole folder is a copy of apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/hooks
+// TODO: avoid code duplication
```

Comment:
TODO: remove code duplication between `packages/launch/src/hooks` and `apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/hooks`